Source/Core/Core/DSP/Jit/x64/DSPJitArithmetic.cpp

```diff
@@ -23,7 +23,7 @@ void DSPEmitter::clr(const UDSPInstruction opc)
 {
   u8 reg = (opc >> 11) & 0x1;
   //	dsp_set_long_acc(reg, 0);
-  MOV(64, R(RAX), Imm64(0));
+  XOR(32, R(EAX), R(EAX));
```

Comment:
Does this clear the upper half of the register as well?

Reply:
Yes.